rtshell/rtcon.py

```diff
@@ -134,6 +134,12 @@ def property_callback(option, opt, option_value, parser):
         print('{0}: No ports specified.'.format(os.path.basename(sys.argv[0])),
                 file=sys.stderr)
         return 1
+
+    if len(args) < 2:
```

Comment:
A connection is allowed to have a single port. rtcon must allow this use case. For example, the DDS port for RTM relies on using rtcon to connect a single port to a topic.